lua/default-config.lua

```diff
@@ -1250,6 +1250,19 @@ lvim.lang = {
       },
     },
   },
+  ps1 = {
+    formatter = {},
```

Comment:
it should be `formatters` i think

Reply:
Good point. I copy pasted the configuration above which (also) has the error. I will fix both.